bin/gluegun

```diff
@@ -20,13 +20,14 @@ opt_parser.parse!
 
 case ARGV[0]
 when "generate"
-  #puts "call start on options #{options.inspect}"
   if ARGV[1].empty?
     puts opt_parser
   else
-    Gluegun::Gluegun.gluegun_generate_pages ARGV[1]
-    Gluegun::Gluegun.gluegun_generate_index ARGV[1]
+    link = ARGV[1].dup
+    site_config_file = Gluegun::Gluegun.raw(link)
```

Comment:
I feel we need not call this at the top level here. Call this internally inside generate function. If you want to call it here, then we have to have an overall validation like function instead to check if the URL / local file is usable.